tests/manage/pv_services/test_pvc_disruptive.py

```diff
@@ -228,12 +228,22 @@ def pods_creation(self, pvc_objs, pod_factory, interface):
 
         # Create one pod using each RWO PVC and two pods using each RWX PVC
         for pvc_obj in pvc_objs:
+            if pvc_obj.volume_mode:
```

Comment:
isn't volume_mode 'Block' or 'FS',  how does this return True or False?

Reply:
We are not passing volumeMode in yaml if it is filesystem. So default value is set as None. Value is specified only if it is block. So it will be either 'Block' or None. I will change this implementation because pvc_obj.volume_mode will have value None for filesystem.

Reply:
Done